tests/update.rs

```diff
@@ -17,6 +19,27 @@ fn mock_substrate_stdlib() -> Vec<u8> {
     assets::read_bundle_from_project("testing-substrate-stdlib", "testing-substrate-stdlib")
 }
 
+macro_rules! move_script_params {
+    ($($param:expr),*) => {
+        vec![
+            $(bcs::to_bytes($param).unwrap()),*
+        ]
+    }
+}
+
+macro_rules! move_script_transaction {
```

Comment:
I am not sure if I got this right :thinking: Isn't that the case with every function and every macro?! You always need to know what parameters to be passed and what the method/macro is doing for you? :upside_down_face:

Reply:
Sure, but jumping to macro definition is not implemented in all IDEs, so sometimes doing things inline makes the code easier to read since all the context is within the test. Repeated code in tests doesn't impact performance, since it's only in tests

If a macro has a nice name (like in this case), then just using better names for macro arguments helps, see the comment above

Reply:
Added something nice for you :wink: :+1: